Code-the-Dream-School/bb-practicum-team-3-front
# added an image gallery to Home page

## Description

Added a responsive image gallery to Home page, made the images clickable, so each image is a link to specific search result 

## Related Issue


## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


### After


![image](https://user-images.githubusercontent.com/28917654/233898994-e1b29108-7d79-4a80-a63f-98451a557472.png)



## Testing Steps / QA Criteria

Choose one of the popular destination you are interested in
Click the image
It should bring you to the search result page where you can see the hotels from the chosen destination
